storage/sealer/ffiwrapper/sealer_test.go

```diff
@@ -412,6 +412,16 @@ func TestSealPoStNoCommit(t *testing.T) {
 	fmt.Printf("EPoSt: %s\n", epost.Sub(precommit).String())
 }
 
+func TestMain(m *testing.M) {
+	//setup()
+	// Here it no-longer is bound to 30s but has 1m30s for the whole suite.
```

Comment:
What 30s timer? Is there some test time limit I'm not aware of? Where is that defined?

Reply:
Go tests fail after 30 seconds by default. When implementing TestMain, you get 1m30s for the whole setup + test, and the 30s test-body window still applies.